plugins/woocommerce/src/Internal/Admin/ShippingLabelBanner.php

```diff
@@ -36,40 +39,26 @@ public function __construct() {
 	 */
 	private function should_show_meta_box() {
 		if ( ! $this->shipping_label_banner_display_rules ) {
-			$jetpack_version   = null;
-			$jetpack_connected = null;
-			$wcs_version       = null;
-			$wcs_tos_accepted  = null;
-
-			if ( defined( 'JETPACK__VERSION' ) ) {
-				$jetpack_version = JETPACK__VERSION;
-			}
+			$dotcom_connected = null;
+			$wcs_version      = null;
 
 			if ( class_exists( Jetpack_Connection_Manager::class ) ) {
-				$jetpack_connected = ( new Jetpack_Connection_Manager() )->has_connected_owner();
+				$dotcom_connected = ( new Jetpack_Connection_Manager() )->has_connected_owner();
 			}
 
-			if ( class_exists( '\WC_Connect_Loader' ) ) {
-				$wcs_version = \WC_Connect_Loader::get_wcs_version();
-			}
-
-			if ( class_exists( '\WC_Connect_Options' ) ) {
-				$wcs_tos_accepted = \WC_Connect_Options::get_option( 'tos_accepted' );
+			if ( class_exists( '\Automattic\WCShipping\Utils' ) ) {
+				$wcs_version = \Automattic\WCShipping\Utils::get_wcshipping_version();
```

Comment:
Should we rename this? I ask because "wcs" is used internally for the connect server, and could be misunderstood as "WC Services" (I know, old school folks 😄)

```suggestion
				$wcshipping_version = \Automattic\WCShipping\Utils::get_wcshipping_version();
```

Reply:
As discussed, I decided to keep the changes as less as possible, so refrained from renaiming, besides wcs can mean wc-shipping as well.